src/ctim/schemas/vocabularies.cljc

```diff
@@ -254,6 +254,7 @@
     "imei"
     "imsi"
     "amp_computer_guid"
+    "orbital_id"
```

Comment:
What is an `orbital_id`? I have a doubt about it being an observable

Reply:
This is the identifier of the orbital node